db/migrate/20150530201009_create_friendships.rb

```diff
@@ -0,0 +1,11 @@
+class CreateFriendships < ActiveRecord::Migration
+  def change
+    create_table :friendships do |t|
+      t.integer :user_id, null: false
+      t.integer :friend_id, null: false
+      t.boolean :status, default: false
```

Comment:
is_confirmed or something might be a better name. It seems like that's what the status being true means.